ambry-api/src/main/java/com/github/ambry/clustermap/ClusterMapChangeListener.java

```diff
@@ -41,4 +42,10 @@ default void onDataNodeRemoved(DataNodeId removedDataNode) {
    */
   default void onRoutingTableChange() {
   }
+
+  /**
+   * Take actions when any change in DataNodeConfig on local node.
+   */
+  default void onDataNodeConfigChange() {
+  }
```

Comment:
This interface is confusing. I suggest making it more generic. This callback should pass the list of data node config to the listener. So whenever there is a change to the data node config, the lister would call this callback.

Reply:
Done.